source/includes/_reserve_page_archive.md

```diff
@@ -0,0 +1,353 @@
+# Reserve Page Archive
+> **Definition**
+
+```
+GET https://demo.ticketswitch.com/f13/reserve_page_archive.v1
+```
+
+This API call should be used in preference to the [status](#status) API call.
+It is faster than the status API call.
+It returns an exact replica of the original reservation response.
```

Comment:
This burst of short sentences is not great style. I thought you wrote it better in the pyticketswitch docs:
https://github.com/ingresso-group/pyticketswitch/blob/f2df8a0a4ad2c13231ebc0bc509885aa2f9438ac/pyticketswitch/client.py#L1399-L1401